runtime/commonMain/src/kotlinx/serialization/Migrations.kt

```diff
@@ -141,24 +141,20 @@ public inline fun <reified T : Any> StringFormat.stringify(objects: List<T>): St
     replaceWith = ReplaceWith("stringify(map)")
 )
 public inline fun <reified K : Any, reified V : Any> StringFormat.stringify(map: Map<K, V>): String =
-    encodeToString(MapSerializer(serializersModule.getContextualOrDefault<K>(), serializersModule.getContextualOrDefault<V>()), map)
+    encodeToString(MapSerializer(serializersModule.serializer<K>(), serializersModule.serializer<V>()), map)
 
-@ImplicitReflectionSerializer
-@OptIn(UnsafeSerializationApi::class)
 @Deprecated(
-    level = DeprecationLevel.WARNING,
+    level = DeprecationLevel.ERROR,
     message = "This method is deprecated for removal. Please use reified getContextualOrDefault<T>() instead",
```

Comment:
Are you sure this is the correct message given `ReplaceWith("getContextual(klass) ?: klass.serializer()")` ?